dsl/pace/dsl/stencil.py

```diff
@@ -224,6 +229,7 @@ def __init__(
         )
         numpy_stencil_config = StencilConfig(
             backend="gtc:numpy",
+            cache_dir=".gt_cache",
```

Comment:
Wouldn't this have to match with the rank as well? This is supposed to check if numpy give you the same answer.

Reply:
Ah, right. This should probably match the one in the `stencil_config` passed for consistency!